packages/shared/src/hooks/usePostById.ts

```diff
@@ -124,10 +122,9 @@ const usePostById = ({ id, options = {} }: UsePostByIdProps): UsePostById => {
       post: post?.post,
       relatedCollectionPosts: post?.relatedCollectionPosts,
       isError,
-      isFetched,
-      isPostLoadingOrFetching: (isLoading || isFetching) && !isRefetching,
+      isLoading: (isLoading || isFetching) && !isRefetching,
```

Comment:
I meant this can now just be `isLoading`?

Reply:
Technically you would say so, just introduced many side effects of the header not showing after initial load though 🫠

Reply:
<img width="431" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/d1a57086-8447-499c-8e88-c050595f4dee">

console log of state before/after

Reply:
Changed - retested now it worked 🫠
Maybe it was cache or just me